packages/salesforcedx-apex-debugger/package.json

```diff
@@ -13,8 +13,8 @@
     "@salesforce/salesforcedx-utils-vscode": "40.8.0",
     "faye": "1.1.2",
     "request-light": "0.2.1",
-    "vscode-debugadapter": "1.22.0",
-    "vscode-debugprotocol": "1.22.0"
+    "vscode-debugadapter": "1.23.0",
```

Comment:
Upgraded so I can use source location in a VS Code debug adapter Output event.
`https://github.com/Microsoft/vscode-debugadapter-node/issues/94`